corehq/apps/registration/views.py

```diff
@@ -266,7 +266,7 @@ def page_context(self):
                 FeatureMeta(_("Guaranteed 1:1 support"), False, None),
                 FeatureMeta(_("Dedicated customer success"), False, None),
             ],
-            "pricing_url": "#",
+            "pricing_url": "//dimagi.com/commcare/pricing",
```

Comment:
Is this deliberately not `https://dimagi.com/commcare/pricing`? Just curious. Also why not put this url directly in the HTML?

Reply:
`//` defaults to whatever protocol the host server is using. in this case it's `https`